tools/cli/domainUtils.go

```diff
@@ -162,6 +162,14 @@ var (
 			Name:  FlagReason,
 			Usage: "Reason for the operation",
 		},
+		cli.StringFlag{
+			Name:  FlagFailoverTypeWithAlias,
+			Usage: "Domain failover type. Default value: force. Options: [force,grace]",
+		},
+		cli.IntFlag{
+			Name:  FlagFailoverTimeoutWithAlias,
+			Usage: "[Optional] Domain failover timeout in seconds. Default value: 60 seconds",
```

Comment:
you can add a Value in the flag as default, which can simplify your code and get rid of `defaultFailoverTimeout`

Reply:
Good to know. Updated it